lib/measures/ConstrainSupplyAirTemperatureResetVerification/measure.py

```diff
@@ -0,0 +1,276 @@
+                    "result_md_name": "report_summary.md",
+                    "report_format": "markdown",
+                },
+                "Payloads": {"reporting_obj": "$"},
+                "Next": "report_cases",
+            },
+            "report_cases": {
+                "Type": "MethodCall",
+                "MethodCall": "Payloads['reporting_obj'].report_multiple_cases",
+                "Parameters": {},
+                "Payloads": {},
+                "Next": "Success",
+            },
+            "Success": {
+                "Type": "MethodCall",
+                "MethodCall": "print",
+                "Parameters": [
+                    "Congratulations! The supply air temperature reset verification workflow is executed with expected results and no error!"
+                ],
+                "End": "True",
+            },
+            "Report Error in workflow": {
+                "Type": "MethodCall",
+                "MethodCall": "logging.error",
+                "Parameters": ["Something is wrong in the workflow execution"],
+                "End": "True",
+            },
+        }
+
+        return workflow
+
+    def get_verification_case(
+        self,
+        idf_file_path,
+        output_dataset_path,
+        supply_outlet_node,
+        air_loop,
+        design_zone_cooling_air_temp,
+    ):
+        supply_air_temperature_reset_verification_case = {
+            "no": 1,
+            "run_simulation": False,
+            "expected_result": "pass",
+            "simulation_IO": {"idf": idf_file_path, "output": output_dataset_path},
+            "datapoints_source": {
+                "idf_output_variables": {
+                    "T_sa_set": {
+                        "subject": supply_outlet_node.nameString(),
+                        "variable": f"{air_loop.nameString()} Supply Outlet Temperature",
+                        "frequency": "detailed",
+                    }
+                },
+                "parameters": {"T_z_coo": design_zone_cooling_air_temp},
+            },
+            "verification_class": "SupplyAirTempReset",
+        }
+
+        return {"cases": [supply_air_temperature_reset_verification_case]}
+
+    def run(
+        self,
+        model: openstudio.model.Model,
+        runner: openstudio.measure.OSRunner,
+        user_arguments: openstudio.measure.OSArgumentMap,
+    ):
+        """
+        Define what happens when the measure is run
+        """
+        super().run(model, runner, user_arguments)
+
+        if not (runner.validateUserArguments(self.arguments(model), user_arguments)):
+            return False
+
+        air_loop_name = runner.getStringArgumentValue("air_loop_name", user_arguments)
+        design_zone_cooling_air_temp = runner.getDoubleArgumentValue(
+            "design_zone_cooling_air_temp", user_arguments
+        )
+        idf_file_path = runner.getStringArgumentValue("idf_path", user_arguments)
```

Comment:
@jslane-h - I feel like this is a bit confusing since this is an OpenStudio measure. Also, because we're not running the simulation using ConStrain I think this could be left blank in the verification case. If so, we could get rid of that argument. Have you tried it? If no, could you please give it a shot?